app/assets/stylesheets/darkswarm/typography.css.scss

```diff
@@ -114,7 +114,7 @@ ul.bullet-list, ul.check-list {
   }
 
   li:before {
-    content: "";
+    content: "•";
```

Comment:
how do you even make your keyboard print something like this? :-)
it makes me think about when I was 10 and coding on my commodore64, you could draw by printing special characters like this!

Reply:
![](https://media.giphy.com/media/d3JIxO0bLk27ZSus/giphy.gif)